src/Bicep.Core.Samples/Files/AKS_LF/main.ir.bicep

```diff
@@ -0,0 +1,131 @@
+// mandatory params
+//@[00:1520) ProgramExpression
+param dnsPrefix string
+//@[00:0022) ├─DeclaredParameterExpression { Name = dnsPrefix }
+param linuxAdminUsername string
+//@[00:0031) ├─DeclaredParameterExpression { Name = linuxAdminUsername }
+param sshRSAPublicKey string
+//@[00:0028) ├─DeclaredParameterExpression { Name = sshRSAPublicKey }
+
+@secure()
+//@[00:0046) ├─DeclaredParameterExpression { Name = servcePrincipalClientId }
+param servcePrincipalClientId string
+
+@secure()
+//@[00:0051) ├─DeclaredParameterExpression { Name = servicePrincipalClientSecret }
+param servicePrincipalClientSecret string
+
+// optional params
+param clusterName string = 'aks101cluster'
+//@[00:0042) ├─DeclaredParameterExpression { Name = clusterName }
+//@[27:0042) | └─StringLiteralExpression { Value = aks101cluster }
+param location string = resourceGroup().location
+//@[00:0048) ├─DeclaredParameterExpression { Name = location }
+//@[24:0048) | └─PropertyAccessExpression { PropertyName = location }
+//@[24:0039) |   └─FunctionCallExpression { Name = resourceGroup }
+
+@minValue(0)
+//@[00:0055) ├─DeclaredParameterExpression { Name = osDiskSizeGB }
```

Comment:
Good point; the range technically shows this (`[00:0055)`), but this isn't clear at all.

I'm using the `SourceSyntax` property to map back to a position in the Bicep file (used for source mapping); not sure if there's much I can do about this, since the decorators are tehcnically part of the param syntax...

Another possible approach here is to just avoid interpolating the IR tree with the Bicep file (and **just** write the tree to file); not sure if that would make changes easier or more difficult to review.

Reply:
I think the current representation is fine, though I'm guessing everyone encountering a `.ir.bicep` file for the first time will have to remind themselves that statements technically start with the first decorator, not the keyword. 

One alternative would be to add some logic on the `SourceSyntax` => position mapping code to skip over the leading nodes if the source syntax is of type `DecorableSyntax`, but it's probably best to keep the mapping simple and straightforward.